### ISSUE TITLE
Extension no longer working after update

### ISSUE THREAD

**Issue by Ares9323** [amir9480/vscode-cpp-helper#41]
Hi, I've just updated the extension and NONE of the command is working..

I tried uninstalling and resetting the settings.
I Tested: 
- Create implementation (nothing happens)
- Create implementation here (nothing happens)
- Copy Implementation (nothing happens)
- Create Header Guard (nothing happens)

I rolled back to previous version, If you want me to test please tell me and I will update again!
I really use this extension everyday!

**Comment by amir9480** [comment]
Hi @Ares9323

Please check your console as described [here](https://github.com/amir9480/vscode-cpp-helper/issues/1#issuecomment-603793422) step by step.

**Comment by Ares9323** [comment]
Hi, I already checked the console and literally nothing is posted (both launching with right click or shortcut).

https://user-images.githubusercontent.com/46932745/135876983-dc4d70a0-4b7b-4eb1-9168-77084822434e.mp4

**Comment by amir9480** [comment]
@Ares9323 
I'm not sure why this happening
Can you uninstall and install the extension again?
Did you try using the extension on another machine?
If possible please send me your source code, it can be a regex problem with your code.

**Comment by Ares9323** [comment]
Probably it's a regex problem, I tried to install vscode in a new virtual machine and I'm encountering the same problem, also the "copy implementation" doesn't work so it shouldn't be a path problem.

Here are the two files I used in the video, but it doesn't work in all the project (Previous version still works perfectly

(I had to add .txt to the extension because .cpp and .h are not accepted)

[DungeonRoom.cpp.txt](https://github.com/amir9480/vscode-cpp-helper/files/7280068/DungeonRoom.cpp.txt)
[DungeonRoom.h.txt](https://github.com/amir9480/vscode-cpp-helper/files/7280069/DungeonRoom.h.txt)
y)

**Comment by NoevilMe** [comment]
it looks like this happens if there is some comment in header file.  // or /**/.
vsc 1.62.0 insider.

**Comment by swq123459** [comment]
same

**Comment by maidamai0** [comment]
same for me

**Comment by studiosoho** [comment]
> it looks like this happens if there is some comment in header file. // or /**/. vsc 1.62.0 insider.

That worked for me.
Removed all comments from header file and the extension worked.

**Comment by Ares9323** [comment]
> > it looks like this happens if there is some comment in header file. // or /**/. vsc 1.62.0 insider.
> 
> That worked for me. Removed all comments from header file and the extension worked.

Yes, we know that's the problem, but removing commenti in the header Is NOT an acceptable soluzione, I'd rather use the older version

**Comment by Ares9323** [comment]
> > it looks like this happens if there is some comment in header file. // or /**/. vsc 1.62.0 insider.
> 
> That worked for me. Removed all comments from header file and the extension worked.

Yes, we know that's the problem, but removing comments in the header Is NOT an acceptable solution, I'd rather use the older version

**Comment by Ares9323** [comment]
Thanks for the update, it seems to work now!

**Comment by Ares9323** [comment]
Edit: It's working but only on some files, for example in these it's still having the same problem:
[Test.zip](https://github.com/amir9480/vscode-cpp-helper/files/8082762/Test.zip)

Rolled back to 2.2, feel free to contact me if you need anything
